prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -272,7 +284,15 @@ class Hl7Serializer(val metadata: Metadata) {
         val suppressQst = hl7Config?.suppressQstForAoe ?: false
         val suppressAoe = hl7Config?.suppressAoe ?: false
         // and we have some fields to suppress
-        val suppressedFields = hl7Config?.suppressHl7Fields?.split(",") ?: emptyList()
+        val suppressedFields = hl7Config
+            ?.suppressHl7Fields
+            ?.split(",")
+            ?.map { it.trim() } ?: emptyList()
+        // or maybe we're going to suppress UNK/ASKU for some fields
+        val blanksForUnknownFields = hl7Config
+            ?.useBlankInsteadOfUnknown
+            ?.split(",")
+            ?.map { it.toLowerCase().trim() } ?: emptyList()
         // start processing
```

Comment:
Adding an additional call to lower case here to match report names, which are in lower case by default.